src/personio_py/models.py

```diff
@@ -289,6 +289,9 @@ def _check_client(self, client: 'Personio' = None) -> 'Personio':
             client.authenticate()
         return client
 
+    def set_client(self, client: 'Personio'):
+        self._client = client
+
```

Comment:
I'm not sure if there is a need for a setter here, this looks like java code to me ^^

We can change `create_attendances()` so that this is not needed

Reply:
Solution proposal: I could change the objects client to be public accessible